com.hma.qlog.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It should be upstreamed.

Reply:
sorry, I don't understand .. I don't speak English
I use a translator .. if you can help me
I will thank you